tdrs-backend/plg/grafana_views/admin_views/admin_tanf_t7.sql

```diff
@@ -0,0 +1,43 @@
+-- AUTOMATICALLY GENERATED FILE ON 2025-05-07 11:16:18
+-- DO NOT EDIT - Your changes will be overwritten
+-- Generated by generate_admin_views.py
+
+-- SQL view for admin_tanf_t7 schema
+-- Transformations applied:
+
+CREATE OR REPLACE VIEW "admin_tanf_t7" AS SELECT "RecordType","CALENDAR_QUARTER","TDRS_SECTION_IND","STRATUM","FAMILIES_MONTH", 
+    data_files.section,
+    data_files.version,
+    data_files.year,
+    data_files.quarter,
+    stt.name AS "STT",                                                         -- Select stt_name from the stts table
+    stt.stt_code AS "STT_CODE",                                                -- Select stt_code from the stts table
+    stt.region_id AS "REGION"                                                  -- Select region from the stts table
+FROM search_indexes_TANF_T7 T7
+INNER JOIN
+        data_files_datafile data_files                                             -- Join with data_files_datafile
+        ON T7.datafile_id = data_files.id                              -- Join condition
+    INNER JOIN (
+        SELECT
+            stt_id,                                                                -- Select stt_id
+            section,                                                               -- Select section
+            year,                                                                  -- Select fiscal_year
+            quarter,                                                               -- Select fiscal_quarter
+            MAX(version) AS version                                                -- Get the maximum version for each group
+        FROM
+            data_files_datafile                                                    -- Subquery table
+        GROUP BY
+            stt_id, section, year, quarter                                         -- Group by columns
+    ) most_recent
+        ON data_files.stt_id = most_recent.stt_id
+        AND data_files.section = most_recent.section
+        AND data_files.version = most_recent.version
+        AND data_files.year = most_recent.year
+        AND data_files.quarter = most_recent.quarter
+    INNER JOIN
+        stts_stt stt                                                               -- Join with the stts table (aliased as stt)
+        ON data_files.stt_id = stt.id                                              -- Join condition to match stt_id
+    WHERE
```

Comment:
another reminder to self, stts that have malformed stratum files also have several incomplete records that must be filtered out in views. for now i will handle this in the dashboard.